src/quo2/components/counter/step/view.cljs

```diff
@@ -0,0 +1,23 @@
+(ns quo2.components.counter.step.view
+  (:require
+    [quo2.components.counter.step.style :as style]
+    [quo2.components.markdown.text :as text]
+    [react-native.core :as rn]
+    [utils.number :as utils-number]))
+
+(defn step
+  [{:keys [type accessibility-label override-theme in-blur-view?]} value]
+  (let [type                (or type :neutral)
+        value               (utils-number/parse-int value)
+        label               (str value)
+        size                (count label)
+        accessibility-label (or accessibility-label (keyword (str "step-" label)))]
```

Comment:
I think it's preferable just to change
`(keyword (str "step-" label)` to something like `:step-counter`
because it can be set with `accessibility-label` prop anyway

Reply:
It's worth mentioning accessibility labels don't need to be dynamic in most cases, since Appium has decent query selectors. This is mentioned in the guidelines too https://github.com/status-im/status-mobile/blob/develop/doc/new-guidelines.md#accessibility-labels.

My personal approach so far has been to only add dynamic accessibility labels like `(str "step-" label)` if they request during review.